rmd_files/reshaping.Rmd

```diff
@@ -330,15 +339,22 @@ reoffending_real <- Rs3tools::s3_path_to_full_df(
     s3_path = "s3://alpha-r-training/intro-r-extension/adult_reoff_by_prev_off_number_2.csv")
 ```
 
-This data table is in wide format, but if you were to plot the data with `ggplot2`, you would need to put it in long format.
+  a) Examine this data table. Would you describe it as being in wide or long format?
+  b) Is it more 'machine readable' or 'human readable'?
+  c) What, if anything, would you need to do to the data before passing it to be read for plotting by functions from a package like `ggplot2`?
 
- a) Put the data in long format
- b) Remove relevant prefixes
- c) Pass the labels 'quarter' and 'count' to the appropriate arguments to name the columns in your long format table.
+Note, these are real data on reoffending, publicly available, derived from the table [here](https://www.gov.uk/government/statistics/proven-reoffending-statistics-april-to-june-2021).
+
+---
+
+### Exercise 2 
```

Comment:
Is it worth putting Exercise 2 after the data preview below?

Reply:
yes, good idea, will do!